app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,22 @@
 <!DOCTYPE html>
+<html lang="de">
 <html>
   <head>
-    <title>BelegarbeitRor</title>
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
+    <title>Belegarbeit</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+    <script src="https://use.fontawesome.com/releases/v5.0.8/js/all.js"></script>
```

Comment:
Dafür gibt es ebenfalls entsprechendes Gem. So wird in Rails kein JS oder CSS eingebunden.